.github/workflows/go.yml

```diff
@@ -0,0 +1,25 @@
+name: Go
+
+on:
+  pull_request:
+    branches: [ master ]
+    types: [ opened, synchronize, closed ]
+  push:
+    branches-ignore:
+      - master
```

Comment:
I'm curious if we need both the push and pr rules. It seems like the push rule should cover everything pr does? We might want to run on push to master too since it can be possible to pass on a branch and fail in master

Reply:
Oh... so you think merging the PR would trigger a push? tbh, I don't know. What I'll do is remove everything related to PR and we will add them back if something does not work as expected 🤞

Reply:
i would just copy from another project like: https://github.com/ent/ent/blob/master/.github/workflows/ci.yml

Reply:
We'll see 😃 what pull_request is actually useful for... now that I have disabled it.

Reply:
let's give it a shot!